### PR TITLE
fixed sf3-incompatibility

### PR DESCRIPTION
**Pull request by cwhartl**

### CODE REVIEW

File: Twig/Extension/AliDatatableExtension.php [hunk 1]
```diff
@@ -110,7 +112,7 @@ public function datatable($options)
     private function createDeleteForm($id)
     {
         return $this->createFormBuilder(array('id' => $id))
-                        ->add('id', 'hidden')
+                        ->add('id', HiddenType::class)
```
Comment (WickedSilver) [inline Twig/Extension/AliDatatableExtension.php]: ::class does not exist in php 5.4, that is why the build of php 5.4 fails.

**Comment by cwhartl** [comment]
no idea why "testSetDatatableId" fails

**Comment by jverdeyen** [comment]
Any update on this?

**Comment by SulivanDotEu** [comment]
nice!

**Pull request merged**